package.json

```diff
@@ -13,7 +13,7 @@
     "build:es6": "tsc -p src/tsconfig-es6.json",
     "clean": "rm -r amd es lib",
     "pretest": "tslint --project .",
-    "test": "nyc --require ts-node/register mocha test/**/*.test.ts --reporter spec --retries 3 --require 'node_modules/reflect-metadata/Reflect.js' --exit",
+    "test": "nyc --require ts-node/register mocha test/*.test.ts test/**/*.test.ts --reporter spec --retries 3 --require 'node_modules/reflect-metadata/Reflect.js' --exit",
```

Comment:
I've realised that we were not running `test/inversify.test.ts` while running tests. I've fixed that to run all the tests in the root folder as well.

Reply:
They are run in ci.

Reply:
But makes sense to run them locally first